_data/simple-icons.json

```
@@ -7093,6 +7093,27 @@
             "hex": "CC2927",
             "source": "https://de.wikipedia.org/wiki/Datei:Microsoft_SQL_Server_Logo.svg"
         },
+        {
+            "title": "Microsoft SQL Server Management Studio",
+            "hex": "FEB900",
+            "source": "https://aleson-itc.com/wp-content/uploads/2020/03/server-m-studio.svg",
+            "aliases": {
+                "aka": [
+                    "SSMS",
+                    "SQL Server Management Studio"
+                ]
+            }
+        },
+        {
+            "title": "Microsoft SQL Server Management Studio Database",
+            "hex": "FEB900",
+            "source": "https://aleson-itc.com/wp-content/uploads/2020/03/server-m-studio.svg",
+            "aliases": {
+                "aka": [
+                    "SQL Server Management Studio Database"
+                ]
+            }
```

I'm not super happy with this as a source.
Given it's not directly linked to Microsoft, I'd be hesitant using this. Have checked Microsoft's architecture diagram SVGs and they don't appear to be available in those packs. Are you aware of an official Microsoft source we can use?
I'd also remove this redundant alias.

```suggestion
            "source": "https://aleson-itc.com/wp-content/uploads/2020/03/server-m-studio.svg"
```